aks-kubesphere-linux/metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "itemDisplayName": "aks kubesphere linux 1.0",
+    "description": "aks kubesphere linux 1.0",
+    "summary": "Deploy KubeSphere on AKS",
+    "githubUsername": "yaorui",
+    "dateUpdated": "2021-03-05",
+    "type": "QuickStart",
+    "validationType": "Manual",
```

Comment:
why manual?

Reply:
I think the service principals have used in scripts, so use "Manual", I will review https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md and fix it.

Reply:
Ok, if the MSI works (it should) then you can do automated validation - if that doesn't work there are some GEN values for the pipeline that may work (depending on the permissions used) - so we should try those options before resorting to manual validation.